phoenix5-spark/src/it/scala/org/apache/phoenix/spark/PhoenixSparkIT.scala

```diff
@@ -66,29 +66,6 @@ class PhoenixSparkIT extends AbstractPhoenixSparkIT {
     results.toList shouldEqual checkResults
   }
 
```

Comment:
I suggest leaving the disabled insert test in.
Hopefully it will inspire someone to implement the feature.

Reply:
I moved the test to the class PhoenixSparkSqlIT and changed the first step for loading and creating the view.
It seems it won't work when using spark.read and createOrReplaceTempView but I copied the test just in case someone else find out a solution.

Reply:
Thanks